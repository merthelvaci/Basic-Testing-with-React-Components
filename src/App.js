import React from 'react';
import logo from './logo.svg';
import './App.css';
import Header from './components/Header';

const App = () => {
	return (
		<div className="App">
			<Header header="My Header" />
		</div>
	);
};

export default App;
