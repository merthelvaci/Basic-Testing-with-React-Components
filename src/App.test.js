import App from './App';

afterEach(cleanup);

describe('<App /> component tests', () => {
	let tree;
	it('should render App component correctly', () => {
		const { asFragment } = render(<App />);
		expect(asFragment()).toMatchSnapshot();
	});
});
