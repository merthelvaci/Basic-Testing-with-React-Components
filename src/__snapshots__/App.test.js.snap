// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component tests should render App component correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <h1
        data-testid="h1tag"
      >
        My Header
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
